Dockerfile

```diff
@@ -7,14 +7,11 @@ WORKDIR /community-server
 ## Copy the package.json for audit
 COPY package*.json ./
 
-## Verify if there are known vulnerabilities in the dependencies
-RUN npm audit --production --audit-level=high
-
 ## Copy the dockerfile's context's community server files
 COPY . .
 
 ## Install and build the Solid community server (prepare script cannot run in wd)
-RUN npm ci --ignore-scripts && npm run build
+RUN npm ci && npm run build
```

Comment:
You might run into a problem with this one.
See in context: https://github.com/solid/community-server/pull/580#issuecomment-784040104

Reply:
Is that the reason we used to have this in the CI? This was removed in the IDP branch since like Ruben said, it's necessary for bcrypt.

Reply:
It builds and runs fine on my end without `--ignore-scripts`, but it doesn't run with it.

Reply:
There might have been other reasons about the CI change. I don't know.